ios/19/_sentry_armv7.crash

```diff
@@ -0,0 +1,31 @@
+OS Version: iOS 9.3.5 (13G36)
+Report Version: 104
+
+Exception Type: EXC_BAD_ACCESS (SIGSEGV)
+Crashed Thread: 0
+
+Application Specific Information:
+Attempted to dereference null pointer.
+
+Thread 0 name:
+Thread 0 Crashed:
+0   CrashLibiOS                     0x219e52            -[CRLCrashOverwriteLinkRegister crash] (CRLCrashOverwriteLinkRegister.m:53)
+1   CrashLibiOS                     0x219e4f            -[CRLCrashOverwriteLinkRegister crash] (CRLCrashOverwriteLinkRegister.m:49)
```

Comment:
Reporting the same method twice with different lines and the method doesn't call itself recursively results in this being not complete. So this frame should be marked as invalid frame and the result for 32bit here is incomplete.